### PR TITLE
Introduce pyproject toml file

### PR DESCRIPTION
**Pull request by filip-komarzyniec**
Resolves: https://github.com/mthh/jenkspy/issues/30

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools >= 61.0", "Cython"]
```
Comment (filip-komarzyniec) [inline pyproject.toml:2]: According to the [setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html), [python packaging user guide](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) and [setuptools changelog](https://setuptools.pypa.io/en/stable/history.html#v61-0-0) the minimal setuptools version supporting `pyproject.toml` configuration is `61.0`. It further requires `python>=3.7`. 

@mthh what is your stand on still supporting python3.6 (end of life on 2021-12-23)?

**Comment by mthh** [comment]
Thanks for your PR!

File: pyproject.toml (hunk 1 above)
Comment (mthh) [reply pyproject.toml:2]: II think it's now reasonable to drop Python 3.6.

**Comment by filip-komarzyniec** [comment]
Is there anything more to be done as part of the PR? 
Maybe anything to change or improve?

**Comment by mthh** [comment]
My apologies, it's been a hectic week on another project, but I'm trying to get it taken care of by tonight.

**Pull request merged**